lib/takeaway.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_literial_string: true
+
+require 'twilio_client'
+
+class TakeAway
+  attr_reader :basket
+
+  def initialize(output)
+    @basket = []
+    @output = output
+    @menu = {
+      'spring roll' => 0.99,
+      'char sui bun' => 3.99,
+      'pork dumpling' => 2.99,
+      'peking duck' => 7.99,
+      'fu-king fried rice' => 5.99
+    }
+  end
+
+  def read_menu
+    menu.each { |key, value| output.puts "#{key}: £#{value}" }
```

Comment:
I tried to call this method on my new instance and got an error:

```
private method `puts' called for "hello":String (NoMethodError)
```

Reply:
output is $stdout injected so would respond to $stdout. 
I didn't make that clear, sorry